ptrace/debugger/child.py

```diff
@@ -159,6 +159,8 @@ def createChild(arguments, no_stdout, env=None, close_fds=True, pass_fds=()):
         if pass_fds and not close_fds:
             warning("pass_fds overriding close_fds.")
             close_fds = True
+    else:
+        assert not pass_fds, "pass_fds not supported on Windows."
```

Comment:
I also ripped that from `Lib/subprocess.py`, hope that is ok

Reply:
python-ptrace is not supported on Windows. Please remove "if not RUNNING_WINDOWS:" and this code.

Reply:
Done. I got confused by the RUNNING_WINDOWS flag, that I used but did not implement. Pushed requested changes.